src/stores/index.js

```diff
@@ -468,12 +468,15 @@ class RootStore {
 
       if(this.titleProfiles[objectId][profileName].offerings === "custom") {
         const titleOfferings = this.allTitles[objectId].metadata.offerings || {};
-
         this.titleProfiles[objectId][profileName].offeringPermissions =
-          this.titleProfiles[objectId][profileName].offeringPermissions.map(offeringPermission => ({
-            ...(titleOfferings[offeringPermission.offeringKey] || {}),
-            ...offeringPermission
-          }));
+          this.titleProfiles[objectId][profileName].offeringPermissions.map(offeringPermission => {
+            return {
+              displayName: profileName,
+              playoutFormats: titleOfferings[profileName].playout ? Object.keys(titleOfferings[profileName].playout.playout_formats || {}).join(", ") : "",
```

Comment:
`titleOfferings[profileName].playout` could throw an error if `titleOfferings[profileName]` is not present.